docs/source/intro_accuracy_precision.rst

```diff
@@ -31,6 +31,10 @@ for when analyzing DEMs:
 .. figure:: imgs/precision_accuracy.png
     :width: 80%
 
+*Note: sometimes "accuracy" is also used to describe both types of errors, and "trueness" systematic errors, following
+`ISO 5725-1 <https://www.iso.org/obp/ui/#iso:std:iso:5725:-1:ed-1:v1:en>`_. Here, we used accuracy for systematic errors
+as, to our knowledge, it is more common in remote sensing applications."
+
 Source: `antarcticglaciers.org <http://www.antarcticglaciers.org/glacial-geology/dating-glacial-sediments2/precision-
```

Comment:
Source should be moved before the note.

Reply:
I believe a second * is missing no? To make the *Note**appear highlighted (or better ** to make it bold?).

Reply:
Also first TODO below. Should we use figure 1 from your paper?

Reply:
Yes, done for all three suggestions!